drange/lib/DataStructure.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
```

Comment:
The naming is difficult. Maybe module could be still `drange` (and we could add range generation there, if we ever need one), but the arbitrary for the integers in range could be `fromDiscontinuosRange`. How does this sound?

Sorry for the confusion. While I'd like to *keep `jsverify` lib as dependency free as possible*, I'm much more liberal about `jsverify-contrib`. That's why it exists.

`finitemap` is an internal structure (and is mutable to simplify generating functions feature).

I'm ok with both approaches, maybe the first one is easier. Sorry you made unnecessary work :(